package/boot/grub2/grub2-efi/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=grub-efi
+
+include ../common.mk
+
+TAR_OPTIONS:= --transform 's/grub-${PKG_VERSION}/${PKG_NAME}-${PKG_VERSION}/' $(TAR_OPTIONS)
+
+PKG_BUILD_DEPENDS:=grub2-efi/host
+CONFIGURE_ARGS += --with-platform=efi
+HOST_CONFIGURE_ARGS += --with-platform=efi
+HOST_CONFIGURE_ARGS += --with-suffix=efi
+
+define Package/grub2-efi
+$(call Package/grub2/Default)
+DEPENDS:=@EFI_IMAGES +kmod-efi-vars
+TITLE += (with EFI support)
+endef
+
+$(eval $(call HostBuild))
+$(eval $(call BuildPackage,grub2-efi))
+
+include $(INCLUDE_DIR)/kernel.mk
+
+define KernelPackage/efi-vars
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=EFI VARS modules
+  DEPENDS:=@EFI_IMAGES
+  KCONFIG:= \
```

Comment:
These kernel configurations should be included in the subtarget config (``target/linux/x86/64/config-default`` and ``target/linux/x86/generic/config-default``).

To modify the kernel configuration for subtarget, launch ``make CONFIG_TARGET=subtarget kernel_menuconfig``.